### ISSUE TITLE
Fail to build with GCC 10

### ISSUE THREAD

**Issue by eribertomota** [netdiscover-scanner/netdiscover#5]
When building with GCC 10, netdiscover fails and shows:

```
gcc  -g -O2 -fdebug-prefix-map=/PKGS/netdiscover/netdiscover=. -fstack-protector-strong -Wformat -Werror=format-security  -Wl,-z,relro -Wl,-z,now -o netdiscover main.o ifaces.o screen.o fhandle.o misc.o data_reply.o data_request.o data_unique.o -lpthread -lpcap -lnl 
/usr/bin/ld: ifaces.o:./src/ifaces.h:53: multiple definition of `errbuf'; main.o:./src/ifaces.h:53: first defined here
/usr/bin/ld: ifaces.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: ifaces.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: ifaces.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: ifaces.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: ifaces.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: screen.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: screen.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: screen.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: screen.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: screen.o:./src/ifaces.h:53: multiple definition of `errbuf'; main.o:./src/ifaces.h:53: first defined here
/usr/bin/ld: screen.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: misc.o:./src/ifaces.h:53: multiple definition of `errbuf'; main.o:./src/ifaces.h:53: first defined here
/usr/bin/ld: misc.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: misc.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: misc.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: misc.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: misc.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: data_reply.o:./src/data_reply.c:46: multiple definition of `line'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_reply.o:./src/data_reply.c:46: multiple definition of `tline'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_reply.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: data_reply.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: data_reply.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_reply.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_reply.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: data_request.o:./src/data_request.c:46: multiple definition of `line'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_request.o:./src/data_request.c:46: multiple definition of `tline'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_request.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: data_request.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: data_request.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_request.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_request.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: data_unique.o:./src/data_unique.c:47: multiple definition of `line'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_unique.o:./src/data_unique.c:47: multiple definition of `tline'; screen.o:./src/screen.c:55: first defined here
/usr/bin/ld: data_unique.o:./src/screen.h:53: multiple definition of `win_sz'; main.o:./src/screen.h:53: first defined here
/usr/bin/ld: data_unique.o:./src/screen.h:55: multiple definition of `parsable_output'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_unique.o:./src/screen.h:55: multiple definition of `continue_listening'; main.o:./src/screen.h:55: first defined here
/usr/bin/ld: data_unique.o:./src/data_al.h:38: multiple definition of `data_access'; main.o:./src/data_al.h:38: first defined here
/usr/bin/ld: data_unique.o:./src/screen.h:54: multiple definition of `current_network'; main.o:./src/screen.h:54: first defined here
/usr/bin/ld: cannot find -lnl
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:354: netdiscover] Error 1
make[3]: Leaving directory '/PKGS/netdiscover/netdiscover/src'
```

We need help to fix.

**Comment by jpenalbae** [comment]
See first entry in this documentation: https://gcc.gnu.org/gcc-10/porting_to.html

* Fastest way to fix should be compiling with -fcommon
* Proper fix would be using extern in the headers, and define the global in the C file instead.

**Comment by eribertomota** [comment]
Thanks a lot for your help. Fixed by 984225e.

I used -fcommon.

Cheers,

Eriberto
